resources/public/api/conf/2.0/listSelfAssessmentPaymentsAndAllocationDetails.raml

```diff
@@ -0,0 +1,52 @@
+is:
+  - headers.acceptJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+#  - errors.formatStartDate
```

Comment:
Were these commented out because they were getting changed in the spec? I think we can add them in now as they should be updated and correct in the business spec.